src/Microsoft.IdentityModel.Tokens/Base64UrlEncoder.cs

```diff
@@ -177,47 +120,68 @@ public static byte[] DecodeBytes(string str)
             return Decode(str.AsSpan());
         }
```

Comment:
Do we have tests for this `DecodeBytes(string str)`? We use it everywhere but maybe good to have explicit tests like in Base64UrlEncoderTests?

Same questions for `Decode(string arg)` below.